cluster/addons/dns/README.md

```diff
@@ -55,6 +56,11 @@ compatibility, kube-dns supports both names for the time being.  Users should
 avoid creating a namespace named 'svc', to avoid conflicts.  The old name
 format is deprecated and will be removed in a future release.
 
+### Pods
+
+#### A Records
+Pods are assigned a DNS A record in the form of `pod-ip-address.my-namespace.pod.cluster.local`.
```

Comment:
Can you list an example, so it is clear that a pod with IP 1.2.3.4 is `1-2-3-4.ns.pod...` and not `1.2.3.4.ns.pod...`